deployment/mobile_apps/ios/README.md

```diff
@@ -0,0 +1,110 @@
+# Deployment - Mobile Apps - iOS
+
+**NOTE**: This assumes you are using ionic 2 but should translate pretty well
+for any cordova project. It also assumes you have Xcode set up appropriately
+and that you have the login credentials for the mohrlab dev Apple ID.
+
+**ALSO NOTE**: This was written on my second build. It is mostly the same as my
+first build but there may be some differences. Do not take this as gospel and
+check for errors, etc at each step.
+
+I dropped and add the platform each time. You may not need to but I wasn't
+messing with it.
+
+* `ionic platform rm ios`
+* `ionic platform add ios`
+* `STAGE=staging ionic build ios`
+
+(*NOTE*: The set up for stages may be different for your project and you may
+prefer to use the --prod flag)
+
+The last step will likely fail and you will see the following error. (*NOTE*: I
+am pretty sure this can be gotten around by using a `build.json` but I'm not
+sure how far that will get you)
+
+```
+=== BUILD TARGET CAR-OLT OF PROJECT CAR-OLT WITH CONFIGURATION Debug ===
+
+
+Check dependencies
+
+Signing for "CAR-OLT" requires a development team. Select a development team in
+the project editor.
+
+Code signing is required for product type 'Application' in SDK 'iOS 10.2'
+
+
+
+** ARCHIVE FAILED **
+
+
+The following build commands failed:
+
+  Check dependencies
+(1 failure)
+
+Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/Chris/Work/carolt_app/platforms/ios/cordova/build-debug.xcconfig,-workspace,CAR-OLT.xcworkspace,-scheme,CAR-OLT,-configuration,Debug,-destination,generic/platform=iOS,-archivePath,CAR-OLT.xcarchive,archive,CONFIGURATION_BUILD_DIR=/Users/Chris/Work/carolt_app/platforms/ios/build/device,SHARED_PRECOMPS_DIR=/Users/Chris/Work/carolt_app/platforms/ios/build/sharedpch
+```
+
+1. Open Xcode
+1. Open your project. It should be in `/path/to/ionic/project/platforms/ios/`
+   and end with the file extension `.xcodeproj`
+1. In upper left, set device to 'Generic iOS Device',
+1. Under *Xcode > Preferences > Accounts*, add or select mohrlab Apple ID.
+   Select View Details, download or confirm downloaded provisioning profile
+1. Select Done
+1. Close Preferences
+1. Go to <https://developer.apple.com/account/ios/certificate>.
+  * Check to make sure a cert is there for your computer
+  * Under *Devices*, add devices for testing
+  * Under *Provisioning Profiles > Development*, create a new provisioning profile
+  * Under *Provisioning Profiles > Distribution*, create a new provisioning profile
+
+1. Go back to Xcode and select project in left tool bar
+
+  (*NOTE*: The next set of directions are derived from
+  <http://stackoverflow.com/a/39498874/6731497>. These I repeated just about
+  every time I rebuilt the app and either ran it on the device/simulator or
+  archived it for distribution. Some steps may be unnecessary but I am unsure
+  which.)
+
+1. Under *General > Signing > Team* select 'Mark Begale' (this is who the
```

Comment:
Again, maybe generalize these specifics? We will likely change the account in the future.